docs/index.css

```diff
@@ -46,3 +46,35 @@ code {
 .Demo ul {
   margin: var(--space-large);
 }
+
+.Footer {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  height: var(--top-bar-height);
```

Comment:
I'm not sure we need the footer visible at all times. Can it not just be a footer inside of the workspace section?

Reply:
`<footer>` can't go inside `<main>`, so I can't make it a child of `<Workspace>`. I can style it that way, but I thought it looked weird when I tried it initially.

Full width:
![Screen Shot 2022-08-23 at 10 25 11 AM](https://user-images.githubusercontent.com/99676396/186187868-243382ae-2f87-4029-a59d-d2c3f029a660.png)

Workspace width:
![Screen Shot 2022-08-23 at 10 35 15 AM](https://user-images.githubusercontent.com/99676396/186187887-8f66ca1f-9f43-48cf-933a-3e85c5bb38e8.png)

Reply:
eek lol - I like it full width personally